src/pages/guides/index.md

```diff
@@ -256,4 +256,14 @@ hmac.update(raw_request_body)
 if (request.header('x-adobe-signature') !== hmac.digest('base64')) {
   throw new Error('x-adobe-signature HMAC check failed')
 }
-```
\ No newline at end of file
+```
+
+## Quotas
+
+There is an upper limit on the number of registrations that you can create. Following table lists the applicable quotas - 
+
+| Maximum number of registrations  | Grouping level | Description |
+| ------------- | ------------- | ------------- |
+| 2500  | IMS Org  | Maximum of 2500 event registrations (webhook or journal) can be created for an IMS Org  |
+| 10  | IMS Credential (client Id)  | Maximum of 10 event registrations (webhook or journal) can be created for an IMS client-id  |
```

Comment:
We have a way to override this client-id config, so shouldn't we also communicate the same here as well.